posts/2014-03-21-nasa-invites-citizens-collaborate-coastal-flooding-challenge.md

```diff
@@ -0,0 +1,59 @@
+---
+author: ellen-stofan
+categories:
+- arctic
+- climate
+- coastalflooding
+- ecosystem-vulnerability
+- energy-infrastructure
+- foodresilience
+- humanhealth
+- transportation
+- tribal-nations
+- water
+date: '2014-03-21T17:21:46'
+excerpt: |-
+  The U.S. Energy Information Administration (EIA) interactive energy disruption maps combine real-time data feeds from NOAA's National Hurricane Center with more than 20 map layers showing the nation's energy infrastructure and resources. This new tool, available around the clock on the EIA…
+link: https://www.data.gov/climate/nasa-invites-citizens-to-collaborate-on-coastal-flooding-challenge/
```

Comment:
@Jin-Sun-tts Should:
- the main link be `https://www.data.gov/nasa-invites-citizens-to-collaborate-on-coastal-flooding-challenge/` 
- and then `/climate/nasa-invites-citizens-to-collaborate-on-coastal-flooding-challenge/` should be another redirect?

Reply:
I am not very sure which one we should use as main link, I have same question for other pages too. What is the standard/rule we should use here?

Reply:
I think we should put the page at the root and then make all of the other references as the redirects.  Since all of the subfolders are duplicates, it makes sense to put the content outside all of them.